docs/fr/docs/alternatives.md

```diff
@@ -0,0 +1,454 @@
+
+Juste avant de décider de développer **FastAPI**, j'ai trouvé le serveur **APIStar**. Il contenait presque tout ce 
+que je recherchais et avait un beau design.
+
+C'était l'une des premières implémentations d'un framework utilisant les type hints Python pour déclarer les paramètres 
+et les requêtes que j'ai vues (avant NestJS et Molten). Je l'ai trouvé plus ou moins en même temps que Hug. Mais APIStar utilisait le standard OpenAPI.
+
+Il disposait de la validation automatique, sérialisation des données et d'une génération de schéma OpenAPI basée sur les mêmes type hints à plusieurs endroits.
+
+La définition du schéma de corps de requête n'utilisait pas les mêmes type hints Python que Pydantic, il était un peu plus proche de Marshmallow, donc le support de l'éditeur n'était pas aussi bon, mais APIStar était quand même la meilleure option disponible.
+
+Il avait les meilleures performances d'après les benchmarks de l'époque (seulement surpassé par Starlette).
+
+Au départ, il ne disposait pas d'une interface web de documentation automatique de l'API, mais je savais que je pouvais lui ajouter une interface Swagger.
+
+Il avait un système d'injection de dépendances. Il nécessitait un pré-enregistrement des composants, comme d'autres outils discutés ci-dessus. Mais c'était quand même une excellente fonctionnalité.
+
+Je n'ai jamais pu l'utiliser dans un projet complet, car il n'avait pas d'intégration de sécurité, et je ne pouvais donc pas remplacer toutes les fonctionnalités que j'avais avec les générateurs complets basés sur Flask-apispec. J'avais dans mon backlog de projets de créer une pull request pour ajouter cette fonctionnalité.
+
+Mais ensuite, le projet a changé d'orientation.
+
+Il ne s'agissait plus d'un framework web API, le créateur devant se concentrer sur Starlette.
+
+Maintenant, APIStar est un ensemble d'outils pour valider les spécifications OpenAPI, et non un framework web.
+
+!!! info
+    APIStar a été créé par Tom Christie. Le même gars qui a créé :
+
+    * Django REST Framework
+    * Starlette (sur lequel **FastAPI** est basé)
+    * Uvicorn (utilisé par Starlette et **FastAPI**)
+
+!!! check "Inspired **FastAPI** to"
+    Exister.
+
+    L'idée de déclarer plusieurs choses (validation des données, sérialisation et documentation) avec les mêmes types Python, tout en offrant un excellent support pour les éditeurs, était pour moi une idée brillante.
+    
+    Et après avoir longtemps cherché un framework similaire et testé de nombreuses alternatives, APIStar était la meilleure option disponible.
+
+    Puis APIStar a cessé d'exister en tant que serveur et Starlette a été créé, et a constitué une meilleure base pour un tel système. Ce fut l'inspiration finale pour construire **FastAPI**.
+
+    Je considère **FastAPI** comme un "successeur spirituel" d'APIStar, tout en améliorant et en augmentant les 
+fonctionnalités, le système de typage et d'autres parties, sur la base des enseignements tirés de tous ces outils 
```

Comment:
The linebreak with no indenting breaks the formatting
![image](https://user-images.githubusercontent.com/16785985/115157028-69bdaf00-a087-11eb-9543-c6db748d24a9.png)